exercises/starknet/README.md

```diff
@@ -1,26 +1,43 @@
 # Starknet Smart Contracts
 
-A Starknet contract can be created by annotating a Cairo module with the `#[contract]` attribute
+A Starknet contract can be created by annotating a Cairo module with the `#[starknet::contract]` attribute.
 
 Starknet contract don't require a `main` function.
-Functions in the contract module may be annotated as a `#[view]` or an `#[external]` function. External/View functions can be called by the users of Starknet, and by other contracts. The functions without these annotations are internal and cannot be accessed by users nor by other contract.
+Functions in the contract module may be annotated as `#[external(v0)]` function. External functions can be called by the users of Starknet, and by other contracts.
+
+ The functions without these annotations are internal and cannot be accessed by users nor by other contract.
```

Comment:
> The functions without these annotations are internal and cannot be accessed by users nor by other contract.

The functions outside of these blocks are internal and cannot be accessed by users nor by other contract.